File: 3019. Number of Changing Keys/Solution.py
class Solution:
    def countKeyChanges(self, s: str) -> int:
        s = s.lower()
        curr_key = s[0]
        count = 0
        for i in range(1,len(s)):
            if curr_key!=s[i]:
                count+=1
            curr_key = s[i]
        return count

File: 3019. Number of Changing Keys/Solution.js
/**
 * @param {string} s
 * @return {number}
 */
var countKeyChanges = function(s) {
    s = s.toLowerCase();
    let curr_key = s[0];
    let count = 0;
    for(let i = 1;i<s.length; i++){
        if (curr_key!=s[i]){
            count++;
        }
        curr_key = s[i];
    }
    return count;
};
